e2e/cypress/integration/dashboard-transloadit.spec.ts

```diff
@@ -15,4 +15,28 @@ describe('Dashboard with Transloadit', () => {
 
     cy.get('.uppy-StatusBar-statusPrimary').should('contain', 'Complete')
   })
+
+  it('should close assembly polling when cancelled', () => {
+    cy.get('@file-input').attachFile(['images/cat.jpg', 'images/traffic.jpg'])
+    cy.get('.uppy-StatusBar-actionBtn--upload').click()
+
+    cy.intercept({
+      method: 'GET',
+      url: '/assemblies/*',
+    }).as('assemblyPolling')
+    cy.intercept(
+      { method: 'PATCH', pathname: '/files/*', times: 1 },
+      { statusCode: 204, body: {} },
+    )
+    cy.intercept(
+      { method: 'DELETE', pathname: '/resumable/files/*', times: 1 },
+      { statusCode: 204, body: {} },
+    )
+    cy.wait('@assemblyPolling')
+    cy.get('.uppy-u-reset.uppy-StatusBar-actionCircleBtn[title="Cancel"]').click()
+
+    cy.window().then(({ uppy }) => {
+      expect(Object.values(uppy.getPlugin('Transloadit').activeAssemblies).some(a => a.pollInterval)).to.equal(false)
```

Comment:
Shouldn't it be `null` as you added code to set it to that?
```suggestion
      expect(Object.values(uppy.getPlugin('Transloadit').activeAssemblies).some(a => a.pollInterval)).to.equal(null)
```

Reply:
Also why is it `some` and not `every`?

Reply:
> Shouldn't it be `null` as you added code to set it to that?

No, `some` returns a boolean.

> Also why is it `some` and not `every`?

Hum because if any assembly still has `pollInterval` not falsy, we want the test to fail.

Reply:
I read this wrong, seems okay!